mozilla_vpn/explores/funnel_analysis_granular_steps.explore.lkml

```diff
@@ -0,0 +1,57 @@
+include: "/spoke-default/mozilla_vpn/views/funnel_analysis_granular_steps.view.lkml"
```

Comment:
Thinking through naming - what about `funnel_analysis_detailed` or `funnel_analysis_extra_steps`.

......

Reply:
Changed the file names to funnel_analysis_extra_steps and committed the changes.